x-pack/plugin/esql/qa/testFixtures/src/main/resources/string.csv-spec

```diff
@@ -1373,3 +1373,47 @@ emp_no:integer | first_name:keyword | e:keyword    | d:keyword
 10030          | null               | null         | null
 10031          | null               | null         | null
 ;
+
+
+repeat#[skip:-8.14.99,reason:new string function added in 8.15]
```

Comment:
We want to move away from version-based skipping to capability-based ones (check for `required_capability` in this file).